extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -55,6 +55,12 @@ class AmpSocialShare extends AMP.BaseElement {
   buildCallback() {
     const typeAttr = user().assert(this.element.getAttribute('type'),
         'The type attribute is required. %s', this.element);
+    const isTypeSupported = typeAttr !== 'system' ||
```

Comment:
I'd write this as
`typeAttr == 'system' ? !!navigator.share ? true`;

for slightly better readability.